### ISSUE TITLE
[DATA] CCDR risk analytics

### ISSUE THREAD

**Issue by matamadio** [GFDRR/rdl-standard#36]
Data produced for the CCDR reports by M. Amadio and A. Essenfelder.

### Countries (9)
- SAR: Pakistan, Nepal, Bangladesh and soon India
- AFR: Mali, BurkinaFaso, Chad, Niger, Nigeria, Ghana, Guinea, Bissau, Mauritania, Ethiopia
- SEA: Cambodia
- LAC: Dominican Republic, OECS

### Data types

- Hazard datasets (global)
  - Floods: Fathom 2019
  - Coastal floods: RWI, Deltares
  - Landslide: ARUP 2018
  - Drought: FAO ASI season 1 and 2 for >30% and 50% area affected
  - Cyclones: GAR15, STORM20, IBTrACSv4
  - Heat stress: VITO WBGT 2016, UTCI 2020
  - Air pollution: ACAG PM2.5 2020

- Exposure datasets (global)
  - Population: [Worldpop 2020](worldpop.org/), [GHS 2020](https://ghsl.jrc.ec.europa.eu/download.php)
  - Built-up: [WSF 2019](https://geoservice.dlr.de/web/maps/eoc:wsf2019), WSF 3D
  - Land cover: [ESA World cover 2020-2021](https://esa-worldcover.org/en)

- Vulnerability models (global):
  - Floods: [JRC](https://publications.jrc.ec.europa.eu/repository/handle/JRC105688) depth-damage curves (built-up); [Jonkman et al. 2008](https://link.springer.com/article/10.1007/s11069-008-9227-5)) mortality function
  - Heat stress: standard index classification ([Blazejczyk et al. 2012](https://link.springer.com/article/10.1007/s00484-011-0453-2))
  - Air pollution: [WHO Air Quality guidelines (2021)](https://apps.who.int/iris/handle/10665/345329)

- Loss (risk)
  - Expected Annual Impact (EAI) as combination of probabilistic hazard models, exposure data and vulnerability functions
  -  Expected Annual Exposure (EAE) as combination of hazard models, exposure data and vulnerability classes

Type | Format | Description
-- | -- | --
Hazard data |  Raster (tiff) | Multiple or individual layers packages on various resolutions - global scale, not clipped for the countries, except Fathom
Exposure data | Raster (tiff) | Individual raster layers
Vulnerability data | Excel file | Noted excel file including all models used in the assessment
Loss data | Excel file, vector (gpkg) | Output including impact for individual RPs and EAI/EAE calculations, indexing, charts, fields legend

All global data uses CRS: 4326

**Comment by matamadio** [comment]
While the discussion about loss schema is ongoing, the other CCDR components can be included.
The question is how to add them. @stufraser1 please check the proposal:

### Global hazard data
The files with global coverage should have just one global entry, instead of multiplicating for each country/extent.
- Already on the catalog, need to add RDLS metadata:
  - [ ] [Flood (FATHOM)](https://datacatalog.worldbank.org/int/search/dataset/0021728/Global-Flood-Hazard--FATHOM-) (Note there is no direct linkage to the zip, we should link to the form request)
  - [ ] [Landslide (ARUP)](https://datacatalog.worldbank.org/int/search/dataset/0037584/Global-landslide-hazard-map)
  - [ ] [Heat stress (VITO)](https://datacatalog.worldbank.org/int/search/dataset/0040194/Global-extreme-heat-hazard)
- To be added on the catalog:
  - [ ] Coastal floods: [Aqueduct](https://www.wri.org/data/aqueduct-floods-hazard-maps); Deltares on NASADEM90m.
  - [ ] Drought: FAO ASI season 1 and 2 for >30% and 50% area affected
  - [ ] Cyclones: GAR15, [STORM Wind speed 2020](https://data.4tu.nl/articles/dataset/STORM_tropical_cyclone_wind_speed_return_periods/12705164), [IBTrACSv4](https://www.ncei.noaa.gov/products/international-best-track-archive)
  - [ ] Air pollution: [ACAG PM2.5 2020](https://sites.wustl.edu/acag/datasets/surface-pm2-5/#V5.GL.02)

### Global exposure data
In the case of exposure, the data are often downloaded as tiles and then processed (country mosaic and clip; resampling in some case). In some cases (WSF3D), we also have the [global extent datasets](https://download.geoservice.dlr.de/WSF3D/files/global/), but in a very large file (>2 Gb).
Thus, differently from Hazard layers, I would provide the national (analysis) extent for these data, uploaded on our catalog and with reference to source models.

### Global vulnerability data
These are global model with minimal regionalisation, so they should be just one global entry for each.

**Comment by matamadio** [comment]
Please check metadata sheet for global hazard layers:

- [Fathom Flood (v2)](https://github.com/GFDRR/rdl-standard/files/12729433/rdls_hzd-FTHv2.xlsx): requires to fix license (commercial or custom) as in #183 
- [STORM Tropical cyclone wind](https://github.com/GFDRR/rdl-standard/files/12729440/rdls_hzd-STORM.xlsx): being added to DDH/RDL (under review)
- Deltares global coastal flood maps: being added to DDH/RDL (under review)

**Comment by stufraser1** [comment]
For Fathom Flood: Only one request: Please remove values from `hazard/event_sets/0/event_count` per this field description "You should only use this field when details of individual events are not included in the RDLS metadata." 
In reality the no. of events in the modelled hazard numbers many more, so including the number of return period layers shown is misleading.

STORM:
- check attributions/0/entity/url for Nadia - includes fathom URL incorrectly
- Same comment as above wrt hazard/event_sets/0/event_count
- URL (https://data.4tu.nl/datasets/504c838e-2bd8-4d61-85a1-d495bdc560c3/3) is missing from resources/0/download_url
- There is an argument for including the different climate model projections as different event_sets. The suggested URL above refers to the general dataset page where there are multiple model runs and median dataset. We could go further to define all, but not necessary for this demonstration maybe -- or define that we're referring to the median with these resources (if thats what you want to refer to).

**Comment by matamadio** [comment]
Thanks Stu, see fixed:

- [Fathom Flood (v2)](https://github.com/GFDRR/rdl-standard/files/12763185/rdls_hzd-FTHv2.xlsx): event count deleted.
- [STORM Tropical cyclone wind](https://github.com/GFDRR/rdl-standard/files/12763237/rdls_hzd-STORM.xlsx): awaiting publishing on DDH. Event count deleted and urls fixed. I have the original download page link in _access url_ and the direct link to zip in DDH to add as _download url_. I specified it's the median of projections (cause I only added that to DDH) but yes, it would be good to include all climate models.

Related to this: I tried to upload a couple of datasets (deltares global flood, STORM) directly to DDH to produce the download link for metadata. However there seems to be issues with data upload from external networks (outside intranet); does it happen to you as well?

**Comment by matamadio** [comment]
Adding for review:

- [ARUP Landslide hazard](https://github.com/GFDRR/rdl-standard/files/12768238/rdls_hzd-ARUP-LS.xlsx)

**Comment by stufraser1** [comment]
For ARUP Landslide:
- I understand why you've added 'convective storm' as trigger, but this might be a limitation we need to address: rainfall is the trigger, but not necessarily only from convective storm.
- IMT is given as `ls_hzd:-`; We need to add a dimensionless landslide hazard index as maybe `ls_HI:-` but this data gave annual frequency so we also need to reference that too, can I suggest: `ann_freq:-`
- 'hazard/event_sets/0/events/0/occurrence/deterministic/thresholds': I think there is a copy-down error here - please confirm. Also remind me what the thresholds refer to please.

**Comment by matamadio** [comment]
>     * I understand why you've added 'convective storm' as trigger, but this might be a limitation we need to address: rainfall is the trigger, but not necessarily only from convective storm.

Yes, agree on principle. The list of triggers is however the same of hazards and processes; proposing to add "extreme rainfall" or "heavy precipitation" to process list. Usefull for floods anyway.

>     * IMT is given as `ls_hzd:-`; We need to add a dimensionless landslide hazard index as maybe `ls_HI:-` but this data gave annual frequency so we also need to reference that too, can I suggest: `ann_freq:-`

It's a composite index that considers frequency and magnitude. The index value does not represent actual frequency in a year (or does it?), thus I'd put just ls_hzd.

Please also see comment I left on [imt.csv PR](https://github.com/GFDRR/rdl-standard/pull/259):

It's not always useful to specify the hazard type in the metric. In some cases it just create unnecessary duplication when the same metric:unit could be used, e.g.
- Why we need to differentiate FL from LS when speaking of flow depth, velocity?
- How tsunami inundation depth is different from FL or LS "flow depth"?
- Why we need to indicate "etc" (extra-tropical cyclones) for strong wind unit?

>   * 'hazard/event_sets/0/events/0/occurrence/deterministic/thresholds': I think there is a copy-down error here - please confirm. Also remind me what the thresholds refer to please.

Thanks, yes copy-down error occurred. Thresholds are the same for all three layers.

| Index value | Hazard class  |
| -- | -- |
| > 0.01 | High |
| 0.001 –   0.01 | Medium |
| < 0.001 | Low |

**Comment by matamadio** [comment]
FAO ASI and ACAG air pollution added to DDH catalogue - awaiting review, then to be added on JKAN
